fabric-item-group-api-v1/src/testmod/resources/fabric.mod.json

```
@@ -1,12 +1,12 @@
 {
   "schemaVersion": 1,
-  "id": "fabric-item-groups-v0-testmod",
-  "name": "Fabric Item Groups (v0) Test Mod",
+  "id": "fabric-item-groups-v1-testmod",
+  "name": "Fabric Item Groups (v1) Test Mod",
```

```suggestion
  "id": "fabric-item-group-api-v1-testmod",
  "name": "Fabric Item Group API (v1) Test Mod",
```
The mod id in the code is already correct btw

---

The mixin config name is also wrong, will all be fixed in my next commit.